src/test/java/com/example/inventoryservice/controller/DishCommandControllerIntegrationTest.java

```diff
@@ -0,0 +1,76 @@
+package com.example.inventoryservice.controller;
+
+
+import com.example.inventoryservice.Util.BasicClassTest;
+import com.example.inventoryservice.entity.Role;
+import com.example.inventoryservice.security.jwt.JwtTokenProvider;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class DishCommandControllerIntegrationTest extends BasicClassTest {
+
+    @Autowired
+    private JwtTokenProvider jwtTokenProvider;
+
+    @Test
+    public void whenCreateDishWithFreshToken_then201() {
+        Map<String, String> recipe = new HashMap<>();
+        recipe.put("name", "cake");
+        Map<Object, Object> dishDto = new HashMap<>();
+        dishDto.put("price", 12);
+        dishDto.put("recipe", recipe);
+        given().header(HEADER, BEARER_PREFIX + jwtTokenProvider.createToken("stuff", Collections.singletonList(Role.KITCHEN_STUFF)))
```

Comment:
You could create a token in a setup method which would be executed before each test. But this is fine as well.